docs/platform/porting/system.md

```diff
@@ -2157,3 +2157,384 @@ $ sensorctl test accelerometer 100 1000 /* enable accelerometer with interval 10
 $ sensorctl test accelerometer 100 1000 0 /* enable accelerometer with interval 100 ms, 1s batch latency and always on option */
 $ sensorctl info accelerometer /* retrieve accelerometer sensor information */
 ```
+
+## Crash Framework
+
+### crash-manager
+
+crash-manager is a tool used to create crash reports when an application gets crashed.
+crash-manager is not usually invoked manually, but automatically by the kernel when the crash occurs. crash-manager generates the crash report and attaches the kernel-provided core, if requested. crash-worker package provides the `99-crash-manager.conf` file, which configures kernel parameters for crash-manager to get invoked automatically on crash. For more information, see [`99-crash-manager.conf`](#99-crash-managerconf).
+
+crash-manager consists of the following options:
+
+Option                | Description
+----------------------|------------------------------------------------------
+-p  --pid=PID         | Specifies the process ID (PID) of the dumped process.
+-u  --uid=UID         | Specifies the real user identifier (UID) of the dumped process.
+-g  --gid=GID         | Specifies the real group identifier (GID) of the dumped process.
+-i  --tid=TID         | Specifies the thread identifier (TID) of the thread that triggered core dump. For more information, see [core(5)](http://man7.org/linux/man-pages/man5/core.5.html).
+-s  --signal=SIG      | Specifies the signal number causing dump.
+-t  --time=TIME       | Specifies the time of dump in seconds since the epoch.
+-l  --live            | Gets the core dump of the running process.
+-k  --kill-after-dump | Kills after dump (only with --live option).
+-r  --print           | Prints the report path to standard output (stdout).
+-h  --help            | Prints the help message.
+
+In addition to handling crashes, crash-manager can create reports from running processes, livedumps. Livedumps contain `core` generated from the running process and are not created automatically but need to be requested explicitly by you. crash-manager needs to be invoked with the `--live` option, to produce the livedump report:
```

Comment:
@mijin85cho,

Thank you for the suggestion.
Please let me know if the following is fine:

"Livedumps contain `core` generated from the running process. The `core` is not created automatically but need to be requested explicitly by you.

I added core to avoid confusion with respect to the subject. Please verify.